packages/ember-intl/addon/helpers/format-date.ts

```diff
@@ -10,17 +9,15 @@ type Options = FormatParameters[1];
 
 interface FormatDateSignature {
   Args: {
-    Named?: Options & { allowEmpty?: boolean };
-    Positional: [Value] | [Value, Options];
+    Named?: Options;
+    Positional: [Value];
```

Comment:
Due to the use of inheritance (`addon/helpers/-format-base.js`), #1633 had accidentally increased the API of `{{format-*}}` helpers.